plugin/web/extensions/sidebar/utils/overrides.ts

```diff
@@ -74,11 +74,14 @@ export const mergeOverrides = (
 
   const needOrderComponents = components
     .filter(c => (c as any).userSettings?.updatedAt)
-    .sort(
-      (a, b) =>
-        ((a as any).userSettings?.updatedAt?.getTime?.() ?? 0) -
-        ((b as any).userSettings?.updatedAt?.getTime?.() ?? 0),
-    );
+    .sort((a, b) => {
```

Comment:
Do you know why we are getting both Date and string values for updatedAt? I feel like we should be able to expect only one type for something like this.

Reply:
It's from post message i think. On mobile all the value become string when doing the sort.

Reply:
Instead of supporting both Date and String, can you just update where we are setting the updatedAt to be turned to a string so we can always expect the same thing?